tasks/docker.py

```diff
@@ -45,9 +48,8 @@ def _show_connection_message(ctx: Context, full_stack_name: str):
     user = connection.user
 
     command = (
-        f'\nssh {user}@{host} "sudo usermod -aG docker {user} && sudo reboot"\n'
+        f'\nssh {user}@{host} --  \'echo "Successfully connected to VM" && exit\' \n'
```

Comment:
Did you check that you can run `docker --context pulumi-{host} container ls`?

Reply:
Yes I tried to copy paste the message and it worked